Blazorade.Bootstrap.Components/ListGroupItem.razor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Blazorade.Bootstrap.Components
+{
+    public partial class ListGroupItem
+    {
+        [Parameter]
+        public bool IsAction { get; set; }
```

Comment:
The `IsAction` property is unnecessary on the basic `ListGroupItem` component, since adding the `action` class to the `&lt;li&gt;` element has no effect. You can try with your browser's dev tools for instance on [these samples](https://getbootstrap.com/docs/4.4/components/list-group/#active-items).

Reply:
It's actually kinda strange. It DOES add one bit of functionality, in that when you click the item, it shades the item darker for the duration of the mouse click. BUT... since it doesn't change the cursor indicating there's any sort of functionality associated with the item, it's sort of useless by itself. I can remove it and let users add it if they have a weird circumstance where they might desire it.

Reply:
![image](https://user-images.githubusercontent.com/7681055/73897895-86404380-4855-11ea-95b9-dcd1386bd7af.png)

Reply:
You are right. I used the wrong class. There is actually a small hover effect, and a mouse down effect. But, since a list group item only contains content and no actionable items, I guess there is no point in enabling that in the component. As you said, if someone wants to create an action out of a normal list group item, they can just add the class on the element.